### ISSUE TITLE
photos but not obs licensing on 'Yes, license my photos so scientists can use my data!'

### ISSUE THREAD

**Issue by loarie** [inaturalist/inaturalist#156]
http://www.inaturalist.org/observations/new?flickr_photo_id=6483815001&project_id=871&taxon_id=54541&welcome=true
'Yes, license my photos so scientists can use my data!' should license obs/sounds/photos not just photos, maybe reword:
Yes, license my content so scientists can use my data!'

**Comment by carrieseltzer** [comment]
I suspect this old issue can be closed, but not sure. Found it when I searched for my own issue about licenses!

**Comment by loarie** [comment]
now says  'Yes, license my photos, sounds & observations so scientists can use my data!' so closing this because it covers sounds
